midi2audio.py

```diff
@@ -37,30 +37,40 @@
 DEFAULT_SOUND_FONT = '~/.fluidsynth/default_sound_font.sf2'
 DEFAULT_SAMPLE_RATE = 44100
 
+
 class FluidSynth():
     def __init__(self, sound_font=DEFAULT_SOUND_FONT, sample_rate=DEFAULT_SAMPLE_RATE):
         self.sample_rate = sample_rate
         self.sound_font = os.path.expanduser(sound_font)
 
-    def midi_to_audio(self, midi_file, audio_file):
-        subprocess.call(['fluidsynth', '-ni', self.sound_font, midi_file, '-F', audio_file, '-r', str(self.sample_rate)])
+    def midi_to_audio(self, midi_file, audio_file, no_log=True):
+        if no_log:
+            subprocess.call(['fluidsynth', '-ni', self.sound_font, midi_file,
+                             '-F', audio_file, '-r', str(self.sample_rate)], stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL)
+        else:
+            subprocess.call(['fluidsynth', '-ni', self.sound_font, midi_file,
+                             '-F', audio_file, '-r', str(self.sample_rate)])
 
     def play_midi(self, midi_file):
-        subprocess.call(['fluidsynth', '-i', self.sound_font, midi_file, '-r', str(self.sample_rate)])
+        subprocess.call(['fluidsynth', '-i', self.sound_font,
+                        midi_file, '-r', str(self.sample_rate)])
+
 
 def parse_args(allow_synth=True):
-    parser = argparse.ArgumentParser(description='Convert MIDI to audio via FluidSynth')
+    parser = argparse.ArgumentParser(
```

Comment:
It would be perfect to separate code formatting and the actual change into two different commits for better clarity.